microsphere-multiactive-spring-cloud-loadbalancer/src/main/java/io/microsphere/multiple/active/zone/spring/cloud/loadbalancer/CustomizedLoadBalancerAutoConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package io.microsphere.multiple.active.zone.spring.cloud.loadbalancer;
+
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.cloud.loadbalancer.annotation.LoadBalancerClientSpecification;
+import org.springframework.cloud.loadbalancer.support.LoadBalancerClientFactory;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Collections;
+
+/**
+ * {@link ZonePreferenceServiceInstanceListSupplier} Auto-Configuration Class
+ *
+ * @author <a href="mailto:warlklown@gmail.com">Walklown<a/>
+ * @since 1.0.0
+ */
+@Configuration(proxyBeanMethods = false)
+@ConditionalOnClass(LoadBalancerClientFactory.class)
+@ConditionalOnProperty(value = "microsphere.customized-loadbalancer.enabled", havingValue = "true")
+public class CustomizedLoadBalancerAutoConfiguration implements InitializingBean {
+
+    @Autowired
+    private LoadBalancerClientFactory loadBalancerClientFactory;
+
+    @Override
+    public void afterPropertiesSet() throws Exception {
+        loadBalancerClientFactory.setConfigurations(Collections.singletonList(
```

Comment:
The `LoadBalancerClientFactory` Bean is a singleton instance in the main Spring ApplicationContext, thus other LoadBalancerClientSpecification beans will be overridden by the customized one.  

Above all, the current class could annotate @LoadBalancerClients, like :
```java
@LoadBalancerClients(ZonePreferenceLoadBalancerClientConfiguration.class)
...
public class ZonePreferenceLoadBalancerAutoConfiguration {
    ...
}
```

Reply:
![image](https://github.com/microsphere-projects/microsphere-multiactive/assets/22697462/73ae99ca-e28c-451d-a38c-d8da3b32f94e)
LoadBalancerClientFactory.setConfigurations will not overwrite the existing LoadBalancerClientSpecification, but will append it. But still taking your suggestion, @LoadBalancerClients looks to be the more elegant implementation. Thanks!